mindsdb/integrations/handlers/postgres_handler/postgres_handler.py

```diff
@@ -267,14 +267,13 @@ def insert(self, table_name: str, df: pd.DataFrame) -> Response:
         columns = df.columns
 
         # postgres 'copy' is not thread safe. use lock to prevent concurrent execution
-        with self._insert_lock:
-            resp = self.get_columns(table_name)
+        resp = self.get_columns(table_name)
 
         # copy requires precise cases of names: get current column names from table and adapt input dataframe columns
         if resp.data_frame is not None and not resp.data_frame.empty:
             db_columns = {
                 c.lower(): c
-                for c in resp.data_frame['Field']
+                for c in resp.data_frame['field']
```

Comment:
Please merge main and change 'field' to 'COLUMN_NAME'

Reply:
changed